MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+linearProgressBar.SecondaryProgressFill = Colors.Red;
+linearProgressBar.SecondaryProgress = 50;
+this.Content = linearProgressBar;
+... 
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## SfCircularProgressBar
+
+<table> 
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[IndicatorInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorInnerRadius)'| markdownify }}</td>
+<td>{{'ProgressThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness for the progress. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
+<tr>
+<td>{{'[IndicatorOuterRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorOuterRadius)'| markdownify }}</td>
+<td>{{'ProgressRadiusFactor'| markdownify }}</td>
+<td>Gets or sets a value that specifies the outer radius factor of the progress.</td></tr>
+<tr>
+<td>{{'[ShowProgressValue](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue)'| markdownify }}</td>
+<td>{{'-'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to show progress value label or not in SfCircularProgressBar.</td></tr>
+<tr>
+<td>{{'[TrackInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_TrackInnerRadius)'| markdownify }}</td>
+<td>{{'TrackThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness of track in circular progress bar. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
+<tr>
+<td>{{'[TrackOuterRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_TrackOuterRadius)'| markdownify }}</td>
+<td>{{'TrackRadiusFactor'| markdownify }}</td>
+<td>Gets or sets a value that specifies the outer radius factor of the track.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'ProgressCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the progress.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'TrackCornerStyle'| markdownify }}</td>
+<td>Gets or sets the value that specifies the corner style of the track.</td></tr>
+<tr>
+<td>{{'-'| markdownify }}</td>
+<td>{{'ThicknessUnit'| markdownify }}</td>
+<td>Gets or sets enum value that indicates to calculate the track and progress thickness either in logical pixel or radius factor.</td></tr>
```

Comment:
<td>Gets or sets the enum value that indicates to calculate the track and progress thickness either in logical pixel or radius factor.</td></tr>

Reply:
done